src/main/kotlin/net/maizegenetics/phgv2/utils/VariantLoadingUtils.kt

```diff
@@ -0,0 +1,281 @@
+package net.maizegenetics.phgv2.utils
+
+import biokotlin.seq.NucSeq
+import biokotlin.util.bufferedReader
+import com.google.common.collect.Range
+import com.google.common.collect.RangeSet
+import com.google.common.collect.TreeRangeSet
+import htsjdk.samtools.SAMSequenceDictionary
+import htsjdk.samtools.SAMSequenceRecord
+import htsjdk.variant.variantcontext.Allele
+import htsjdk.variant.variantcontext.GenotypeBuilder
+import htsjdk.variant.variantcontext.VariantContext
+import htsjdk.variant.variantcontext.VariantContextBuilder
+import htsjdk.variant.variantcontext.writer.Options
+import htsjdk.variant.variantcontext.writer.VariantContextWriterBuilder
+import htsjdk.variant.vcf.*
+import java.io.File
+import java.security.MessageDigest
+import java.security.NoSuchAlgorithmException
+import java.util.*
+import java.util.logging.Logger
+
+private val myLogger = Logger.getLogger("net.maizegenetics.pangenome.db_loading.VariantLoadingUtils")
+
+// We could use the BioKotlin SeqPosition, but it is heavier than we need
+// as it contains a NucSeqRecord to get the chromosome name, and that includes sequence
+// It needs to be Comparable to be used in a RangeSet
+data class Position (val contig: String, val position: Int) : Comparable<Position> {
+    override fun compareTo(other: Position): Int {
+        if (this.contig == other.contig) {
+            return this.position.compareTo(other.position)
+        }
+        return this.contig.compareTo(other.contig)
```

Comment:
As a String comparison, this will put 10 before 1.  Not sure if we want to do anything about that. The TASSEL Position / Chromosome classes did numeral order if possible

Reply:
RIght ... need to think about this. Do you have suggestions?  We can fix it once we have the basics working.